README.md

```diff
@@ -1 +1,49 @@
-# my-portfoliom
\ No newline at end of file
+# my-portfoliom
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+
+Portfolio: setup and mobile version skeleton
+
+> Description the project.
+
+For the first milestone in the process of creating your portfolio website, you will:
+
+set up a new repository and prepare it for development using best practices (e.g. linters).
+build the first 2 sections of the mobile website using the template you chose in the previous step.
+
+<<<<<<< HEAD
+
+![screenshot](./p.png)
+![screenshot](./p1.png)
+
+
+=======
+>>>>>>> 4207069f26825060634c6a9169d4394f5735aec4
```

Comment:
[OPTIONAL] Kindly try to remove lines `16`, `22` and `23` in your README file as it added when there are conflict in your branch